command/results_gotest_parser.go

```diff
@@ -30,6 +30,9 @@ var (
 
 	// Match the end prefix, save PASS/FAIL/SKIP, save the decimal value for number of seconds
 	gocheckEndRegex = regexp.MustCompile(`(PASS|SKIP|FAIL): .*.go:[0-9]+: (\S+)\s*([0-9\.m]+[ ]*s)?`)
+
+	// Match the failing status prefix for go build.
+	goTestFailedStatusRegex = regexp.MustCompile(`FAIL[^:]\s+(\S+) .*`)
```

Comment:
Do you need the part at the end? I'm think the space makes it not match something like "FAIL	github.com/evergreen-ci/evergreen/rest/model	0.097s" but also I'm thinking we don't really need to have very strict matches for what's at the end anyway

Reply:
It does still match something like that because there is a space between the path and the time, but you're right that we don't really need it since \S+ will break at the space anyway, so I'll drop it.

Reply:
Update: actually I've realized that maybe this _isn't_ strict enough, given 1_simple.log
I'm changing this to actually include the words [build failed]. I don't want parse to fail for FAIL github.com/evergreen-ci/evergreen/rest/model 0.097s because that means a test failed and it'll fail in a way we want. I want it to only match for this special case where no tests wil run.